.github/workflows/ci.yml

```diff
@@ -173,6 +173,7 @@ jobs:
           fi
           if [ "${{ steps.filter.outputs.omni_executor_test }}" = "true" ] || [ "$rebuild_parachain" = "true" ] || [ "$rebuild_omni_executor" = "true" ]; then
             run_omni_executor_test=true
+            rebuild_omni_executor=true
```

Comment:
This line is a one-time workaround. Please remember to remove it after this PR merged.